src/services/dynamodb/client_extensions.rs

```diff
@@ -185,6 +192,15 @@ mod tests {
         client
     }
 
+    #[tokio::test]
```

Comment:
```suggestion
    #[test]
```
`tokio::test` is only used to start a tokio runtime, which is needed to execute async operations. This test does not need async operations.